ghdocs/TUTORIAL.md

```diff
@@ -323,24 +507,235 @@ To accomplish this, let's add the following new styles to app.css:
 
 These styles are going to override the default component styling and will help our TextField and Button stand out with some minor positioning adjustments.
 
-Now let's tweak our list a bit. For our purposes, we really want to just have a title and description with the x and checkmark. Since we don't need the paper clip and flag, let's modify one list item to look like the following:
+**Hang on, this part gets a bit tricky**
+
+We really want to tweak our list component for our purposes, we want to have a title and description with the x and checkmark. Since we don't need the paper clip and flag, go ahead and remove all `<div class="ms-ListItem is-selectable"></div>` elements **Except the first one** this will give us one element to work with. 
+
+Inside of the first `.ms-ListItem` remove the elements` <div class="ms-ListItem-action"><i class="ms-Icon ms-Icon--mail"></i></div>` and `<div class="ms-ListItem-action"><i class="ms-Icon ms-Icon--trash"></i></div>`. Then changed `ms-Icon ms-Icon--flag` to `ms-Icon ms-Icon--x` and `ms-Icon ms-Icon--pinLeft` to `ms-Icon ms-Icon--check`. our new list html should look like the following: 
```

Comment:
I'd rephrase to:
"Then change the icons to the x and the check: replace `ms-Icon ms-Icon--flag` with `ms-Icon ms-Icon--x` and `ms-Icon ms-Icon--pinLeft` with `ms-Icon ms-Icon--check`. Our new list HTML should look like the following: "